### PR TITLE
Updated platform support for 4.6.2

### PR DESCRIPTION
**Pull request by rpetrusha**
# Updated platform support for 4.6.2

[Internal Review URL 1](https://review.docs.microsoft.com/en-us/dotnet/framework/get-started/system-requirements?branch=pr-en-us-3758)
[Internal Review URL 2](https://review.docs.microsoft.com/en-us/dotnet/framework/install/guide-for-developers?branch=pr-en-us-3758)
[Internal Review URL 3](https://review.docs.microsoft.com/en-us/dotnet/framework/install/on-windows-10?branch=pr-en-us-3758)
[Internal Review URL 4](https://review.docs.microsoft.com/en-us/dotnet/framework/migration-guide/versions-and-dependencies?branch=pr-en-us-3758)
## Suggested Reviewers

@vivmishra @mairaw

### CODE REVIEW

File: docs/framework/migration-guide/versions-and-dependencies.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Each version of the .NET Framework contains the common language runtime (CLR), t
...
+|4.6.2|4|-   Cryptography enhancements, including support for X509 certificates containing FIS 186-3 DSA, support for persisted-key symmetric encryption, <xref:System.Security.Cryptography.Xml.SignedXml> support for SHA-2 hashing, and increased clarity for inputs to ECDiffieHellman key derivation routines.<br />-   For Windows Presentation Foundation (WPF) apps, soft keyboard support and per-monitor DPI.<br />-   ClickOnce support for the TLS 1.1 and TLS 1.2 protocols.<br />-   Support for converting Windows Forms and WPF apps to UWP apps.||✓  10 Anniversary Update<br /><br /> + 10 November Update <br/> + 10 <br/> + 8.1<br />+ 7|✓  2016<br /><br /> + 2012 R2<br />+ 2012<br />+ 2008 R2 SP1|Use `Release` DWORD:<br /><br /> -   394802 (Windows 10 Anniversary Update)<br />-   394806 (all other OS versions)<br /><br /> (see [instructions](../../../docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md))|
```
Comment (JRAlexander) [inline docs/framework/migration-guide/versions-and-dependencies.md]: For Windows Presentation Foundation (WPF) apps, soft keyboard support, and per-monitor DPI. (missing comma)

**Review by JRAlexander — APPROVED** [review]
One small nit. Otherwise, LGTM.

**Comment by mairaw** [comment]
@rpetrusha FYI, resolved merge conflicts on this one

File: docs/framework/get-started/system-requirements.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ The .NET Framework requires administrator privileges for installation. If you do
 | Windows 10 Fall Creators Update | 32-bit and 64-bit | .NET Framework 4.7.1 | |
 | Windows 10 Creators Update | 32-bit and 64-bit | .NET Framework 4.7 | .Net Framework 4.7.1 |
```
Comment (mairaw) [inline docs/framework/get-started/system-requirements.md]: unrelated to your change: .Net -> .NET

File: docs/framework/install/guide-for-developers.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ Use the following table for quick links, or read further for details. To view th
...
-|**4.6.2**|[NET Framework 4.6.2 Dev Pack](http://go.microsoft.com/fwlink/?LinkId=780617)|[Download page for 4.6.2 web installer](http://go.microsoft.com/fwlink/?LinkId=780597)<br /><br /> [Download page for 4.6.2 offline installer](http://go.microsoft.com/fwlink/?LinkId=780601)|Included in: <br /> Windows 10 Anniversary Edition<br /><br /> You can install on:<br /> Windows 10 <br /> Windows 8.1 and earlier<br /> Windows Server 2012 R2 and earlier<br /> (for a full list, see [system requirements](~/docs/framework/get-started/system-requirements.md))|
+|**4.6.2**|[NET Framework 4.6.2 Dev Pack](http://go.microsoft.com/fwlink/?LinkId=780617)|[Download page for 4.6.2 web installer](http://go.microsoft.com/fwlink/?LinkId=780597)<br /><br /> [Download page for 4.6.2 offline installer](http://go.microsoft.com/fwlink/?LinkId=780601)|Included in: <br /> Windows 10 Anniversary Edition<br /><br /> You can install on:<br /> Windows 10 November Update <br/> Windows 10 <br /> Windows 8.1 and earlier<br /> Windows Server 2012 R2 and earlier<br /> (for a full list, see [system requirements](~/docs/framework/get-started/system-requirements.md))|
```
Comment (mairaw) [inline docs/framework/install/guide-for-developers.md]: Anniversary Edition or Anniversary Update?

File: docs/framework/migration-guide/versions-and-dependencies.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/migration-guide/versions-and-dependencies.md]: Do we really need a comma there?

**Review by mairaw — APPROVED** [review]
LGTM. A few comments.

**Pull request merged**